config/example/config.txt

```diff
@@ -239,3 +239,6 @@ GAMEMODE_ROUNDS_NEEDED 5
 
 ## Default gamemode to auto-switch back to after a round has concluded
 GAMEMODE_DEFAULT extended
+
+## How long the mob will take to chestburst in seconds
```

Comment:
This is wrong. The game will interpret 450 as 45 seconds (because byond does things in ds)

Reply:
It does not, the counter for processing ticks every second

Reply:
Scuffed